python/ray/_private/test_utils.py

```diff
@@ -50,7 +50,6 @@
 )
 from ray.scripts.scripts import main as ray_main
 from ray.util.queue import Empty, Queue, _QueueActor
-from ray.experimental.state.state_manager import StateDataSourceClient
```

Comment:
state manager imports reporter_agent which breaks the test.

Reply:
so we should move it away from the test util (which is how it was supposed to be ...)

Reply:
hmm, why this was a necessary change? Does this mean we couldn't import state_manager from anywhere?

Reply:
test_metrics -> test_utils -> state_manager -> reporter_pb2

And since it is a test, the driver is affected.

It is okay to import this. We should just make sure this is not imported to the core worker code (which makes sense). For most of tests it is fine because we don't import opencensus proto.